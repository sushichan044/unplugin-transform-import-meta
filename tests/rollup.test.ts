import { rollupBuild, testFixtures } from "@sxzz/test-utils";
import path from "node:path";
import { describe } from "vitest";

import Starter from "../src/rollup";

describe("rollup", async () => {
  const { dirname } = import.meta;
  await testFixtures(
    "*.js",
    async (args, id) => {
      const { snapshot } = await rollupBuild(id, [Starter()]);
      return snapshot;
    },
    { cwd: path.resolve(dirname, "fixtures"), promise: true },
  );
});
