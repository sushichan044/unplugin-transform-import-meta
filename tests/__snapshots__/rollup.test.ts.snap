// Vitest Snapshot v1, <https://vitest.dev/guide/snapshot.html>

exports[`rollup > basic.js 1`] = `
"// basic.js
// unplugin-resolve-import-meta injected
const num = 10;

export { num };
"
`;
